ctera/src/main/java/ch/cyberduck/core/ctera/CteraSession.java

```diff
@@ -111,8 +110,7 @@ public CteraSession(final Host host, final X509TrustManager trust, final X509Key
     protected DAVClient connect(final Proxy proxy, final HostKeyCallback key, final LoginCallback prompt, final CancelCallback cancel) {
         final HttpClientBuilder configuration = builder.build(proxy, this, prompt);
         configuration.disableRedirectHandling();
-        configuration.setServiceUnavailableRetryStrategy(new CustomServiceUnavailableRetryStrategy(host,
-                new ExecutionCountServiceUnavailableRetryStrategy(authentication)));
+        configuration.setServiceUnavailableRetryStrategy(new CustomServiceUnavailableRetryStrategy(host, 1, authentication));
```

Comment:
`Math.max(new HostPreferences(host).getInteger("connection.retry"), 1)` ?

Reply:
Do not expect another 302 response from the server.